components/thanos.libsonnet

```diff
@@ -0,0 +1,392 @@
+local t = (import 'kube-thanos/thanos.libsonnet');
+local rc = (import 'thanos-receive-controller/thanos-receive-controller.libsonnet');
+local memcached = (import 'memcached.libsonnet');
+
+local storeShards(shards, cfg) = {
```

Comment:
This should best be moved to kube-thanos, just like the receiveRings. Maybe leave a TODO in code. :)